build.rs

```diff
@@ -21,11 +26,16 @@ fn main() {
 
     let mut content = String::new();
     content.push_str("/// this file is auto-generated by the build script.\n");
-    content.push_str("/// you should never manually change it.\n\n\n");
+    content.push_str("/// you should never manually change it.\n\n");
+    content.push_str("use php_parser_rs::prelude::{Lexer, Parser};\n");
+    content.push_str("use pretty_assertions::assert_str_eq;\n\n");
+    content.push_str("static PARSER: Parser = Parser::new();\n");
+    content.push_str("static LEXER: Lexer = Lexer::new();\n\n");
```

Comment:
can now be sent across threads safely, meaning we can have multiple threads parsing different files using the same parser, previously that was not possible.